### ISSUE TITLE
error when exporting

### ISSUE THREAD

**Issue by jorgebrunal** [mxp100/insomnia-plugin-openapi-export#3]
thank you very much for your plugin!

**Describe the bug**
generates errors when exporting a collection with many requests and folders

**To Reproduce**
I have the following:
- [1] with the example of [petstore example](https://petstore.swagger.io/v2/swagger.json) converted to request works perfect.
- [2] but I have another one that has several request and folders to group the services and it generates this error.

**Expected behavior**
that the openapi file can be generated 😄 

**Screenshots**
1.
![](https://i.imgur.com/liw9rMr.png)

2.
![](https://i.imgur.com/RTliHTD.png)
![](https://i.imgur.com/4bJKJng.png)
![](https://i.imgur.com/d4kvSvC.png)

**Desktop (please complete the following information):**
 - OS: **Mac Catalina (10.15)**
 - Insomnia Version **2023.2.1**
 - Plugin version **2.1.0**

**Additional context**
it is possible (I am not sure) as seen in the screenshots above, that it is possible that some request are called the same, it would have something to do with the error that is generated when exporting?

```
SyntaxError: Unexpected end of JSON input
    at JSON.parse (<anonymous>)
    at /Users/brunaldo/Library/Application Support/Insomnia/plugins/insomnia-plugin-openapi-export/main.js:141:51
    at Array.forEach (<anonymous>)
    at OpenApiExport.getPaths (/Users/brunaldo/Library/Application Support/Insomnia/plugins/insomnia-plugin-openapi-export/main.js:109:18)
    at action (/Users/brunaldo/Library/Application Support/Insomnia/plugins/insomnia-plugin-openapi-export/main.js:303:31)
    at async file:///Applications/Insomnia.app/Contents/Resources/app.asar/index.9c113364.js:696:65571
```

again, thank you very much!
sorry for the bad translation, I don't speak English.

**Comment by sthales** [comment]
I have the same problem

**Comment by mxp100** [comment]
thank you for reporting, i review this problem and please attach you insomnia json file for reproduce.
inside, the function of insomnia itself is used to export first to the json and there was a problem, of course I will try to fix it.
P.S. The problem was that if you specify the content type json and at the same time leave the request body empty (not valid), then the internal export will work with errors.

**Comment by mxp100** [comment]
trouble with this code
 ```
 const insomniaExport = await this.context.data.export.insomnia({
            includePrivate: false,
            format: 'json',
            workspace: this.models.workspace,
});
```
it some situations, exported invalid json string

help me and create new issue on https://github.com/Kong/insomnia/issues

**Comment by shide1989** [comment]
same problem here, did you open an issue yet on https://github.com/Kong/insomnia/issues ?

**Comment by mxp100** [comment]
> same problem here, did you open an issue yet on https://github.com/Kong/insomnia/issues ?

no free time :(, please report about this problem

**Comment by mxp100** [comment]
attach json collection with troubles and add code with trouble

**Comment by mxp100** [comment]
released in 2.1.1

**Comment by mxp100** [comment]
released in 2.1.1
